src/golang/cmd/server/handler/cloud_integration_utils.go

```diff
@@ -125,7 +125,16 @@ func setupTerraformDirectory(dst string) error {
 		return err
 	}
 
-	_, stdErr, err := lib_utils.RunCmd("cp", []string{"-rT", dynamic.TerraformTemplateDir, dst}, "", false)
+	_, stdErr, err := lib_utils.RunCmd(
+		"cp",
+		[]string{
+			"-R",
```

Comment:
should we be using `cp -rT` here? What was the reason behind the change?

Reply:
@hsubbaraj-spiral because Mac's `cp` doesn't support the `-T` flag...I can make a comment on this.